lib/livebook_web/live/session_live.ex

```diff
@@ -767,12 +767,18 @@ defmodule LivebookWeb.SessionLive do
 
   defp evaluated?(cell, data), do: data.cell_infos[cell.id].validity_status == :evaluated
 
-  defp section_to_view(section, data) do
-    %{
-      id: section.id,
-      name: section.name,
-      cell_views: Enum.map(section.cells, &cell_to_view(&1, data))
-    }
+  defp section_views(sections, data) do
+    Enum.map(sections, & &1.name)
+    |> LivebookWeb.Utils.names_to_ids()
```

Comment:
I switched to using `html_id`, because slugs are (at least from what I've seen) part of a URL like `http://www.site.com/news/this-is-a-slug`, and there are (I think) different characters allowed for what's in a URL (without encoding) vs what's allowed in a hash / HTML ID.

I hope that makes sense, but happy to discuss more if you'd like!

Reply:
`html_id` works :)